### ISSUE TITLE
Allow for multiple quote types

### ISSUE THREAD

**Issue by oyeanuj** [GitbookIO/slate-edit-blockquote#9]
Hi @Soreine, wondering if this plugin should include logic similar to `slate-edit-list` allowing for multiple types of lists (ul vs ol)? 

In my usecase, I was using it for blockquotes and pullquotes, and found myself duplicating the logic.

Thoughts?

**Comment by Soreine** [comment]
Can't you use two different instances of the plug-in ? One with the
blockquote type and the other with the pullquote ?

**Comment by oyeanuj** [comment]
@Soreine Yes, that's what I am doing but on noticing how `slate-edit-list` works, that felt like a cleaner solution.

So, I do have a workaround but more of a possible API suggestion.
